aws/resource_aws_cognito_user_pool_client.go

```diff
@@ -218,6 +220,10 @@ func resourceAwsCognitoUserPoolClientCreate(d *schema.ResourceData, meta interfa
 
 	d.SetId(*resp.UserPoolClient.ClientId)
 
+	if err = cognitoUserPoolUICustomizationSet(d, conn); err != nil {
+		return fmt.Errorf("Error setting User Pool Client UI Customization: %s", err)
```

Comment:
Add "Cognito" before "User Pool Client" to be consistent with the error below.

Reply:
```suggestion
		return fmt.Errorf("Error setting Cognito User Pool Client UI Customization: %s", err)
```